Automattic/jetpack
# Add Security Bundle upsell on AAG

#### Changes proposed in this Pull Request:

* Add a Security Bundle Card to the At a Glance
* Supporting changes

| Size | Screenshot |
| ---- | ----------- |
| Mobile | ![Screen Shot 2021-11-29 at 20 08 18](https://user-images.githubusercontent.com/2810519/143990264-26a0ef44-0b9c-4936-871b-ba25da517dd6.png) |
| Medium | ![Screen Shot 2021-11-29 at 20 08 06](https://user-images.githubusercontent.com/2810519/143990284-f98ea915-ae42-458e-bda0-291ec2696f46.png) |
| Desktop | ![Screen Shot 2021-11-29 at 20 07 58](https://user-images.githubusercontent.com/2810519/143990313-a3289a30-cd64-4cb0-ad6c-337e87ab68b9.png) |

#### Jetpack product discussion



pbNhbs-1wI-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






1. Navigate to `/wp-admin/admin.php?page=jetpack#/dashboard` on a site with a security bundle or Complete plan
2. Verify the new dash item matching the above screenshots appears 
3. Add a bundle or Complete plan to the site
4. Verify the Security Bundle upsell does not appear

Broken off from #21866
